src/main/scala/com/fulcrumgenomics/phasing/HapCutToVcf.scala

```diff
@@ -0,0 +1,413 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package com.fulcrumgenomics.phasing
+
+import java.util
+import java.util.NoSuchElementException
+
+import com.fulcrumgenomics.cmdline.{ClpGroups, FgBioTool}
+import dagr.commons.CommonsDef._
+import dagr.commons.io.Io
+import dagr.commons.util.LazyLogging
+import dagr.sopt._
+import htsjdk.samtools.util.CloserUtil
+import htsjdk.variant.variantcontext._
+import htsjdk.variant.variantcontext.writer.{Options, VariantContextWriter, VariantContextWriterBuilder}
+import htsjdk.variant.vcf._
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+
+@clp(
+  description =
+    """
+      |Converts the output of HapCut to a VCF.
+      |
+      |The output of HAPCUT does not include all input variants, but simply those variants that are in phased blocks.
+      |This tool takes the original VCF and the output of HapCut, and produces two VCFs: (1) a VCF containing variants
+      |that were phased, and (2) a VCF containing variants that were not phased, such that all variants in the original
```

Comment:
I'm guessing here, but is this because it's not possible to output unphased genotypes between phased genotypes in VCF?  I.e. if you have variants A-B-c-D with A, B and D phased, you can't stick `c` in the middle?  Otherwise it seems like it would be much nicer to have a single VCF with all variants, with phasing where possible.